package/tests/test_api/test_apps.py

```diff
@@ -530,3 +530,10 @@ def test_api_app_from_json_file_index(self):
         client = TestClient(api_app)
         response = client.get("/")
         assert response.status_code == 200
+
+    def test_api_app_from_fe_json_file_main_api(self):

```

Comment:
Sorry what are we testing with this one?

Reply:
good spot. Sorry this was me trying something for another ticket, I will remove it.